clubs/models.py

```diff
@@ -71,6 +61,104 @@ class Club(models.Model):
         null=True
     )
 
+    occupation = models.TextField(
```

Comment:
I'm not sure we do - if someone wants to list their full position title, things could get long, and we don't know *how* long, so I'm banking on postgres being smart when it comes to variable length text